spec/models/spotlight/reindexing_log_entry_spec.rb

```diff
@@ -0,0 +1,49 @@
+describe Spotlight::ReindexingLogEntry, type: :model do
+  subject { FactoryGirl.build(:reindexing_log_entry) }
+
+  describe 'scope' do
+    before(:all) do
+      # we only want to persist these rows for the duration a given test run...
+      DatabaseCleaner.start
```

Comment:
yeah, i had a note to myself to ask you about that... i'd have expected the spec helper invocation to do the trick, but i noticed that i was running into a problem where the test user was hanging around between runs, and subsequent test runs would fail unless i manually deleted the test user from the db using rails console, or added the `DatabaseCleaner` invocations above.  but i was also surprised by the need for that.  would be happy to get rid of it if possible.  this was only on my laptop, FWIW (though i assume it's not an issue in travis, since it seems like the whole test env is recreated for each run there).